<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>36</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>631</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenWeight</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>731</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tricep Curl</string>
    </property>
   </widget>
   <widget class="QLabel" name="tricepLabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>230</y>
      <width>421</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lb</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tricepCurl">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>260</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>651</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bicep Curl</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bicepCurl">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>360</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="bicepLabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>340</y>
      <width>421</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lb</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>711</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Chest Press</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="chestPress">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>460</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="chestLabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>430</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>1270</y>
      <width>461</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewLog">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>1560</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>View Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dateSetting">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>1110</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Today</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
